apps/openmw/mwgui/travelwindow.cpp

```diff
@@ -158,7 +157,7 @@ namespace MWGui
 
         if (!mPtr.getCell()->isExterior())
             // Interior cell -> mages guild transport
-            MWBase::Environment::get().getSoundManager()->playSound("mysticism cast", 1, 1);
+            MWBase::Environment::get().getWindowManager()->playSound("mysticism cast");
```

Comment:
Why is casting Mysticism spells immune from underwater sound effects, but not other types of spells?